doc/modules/mod_muc_commands.md

```diff
@@ -0,0 +1,47 @@
+# MongooseIM's multi-user chat commands set
+
+## Purpose
+This is a set of commands, which provide actions connected with multi-user chat features.
+
+## Configuration
+This module contains command definitions which are loaded when the module is activated.
+There are no options to be provided, therefore the following entry in config file is sufficient:
+
+```
+{mod_muc_commands, []}
+```
+
+## Commands
+This file consists of [commands definitions](http://mongooseim.readthedocs.io/en/latest/modules/mod_commands/).
+This module defines following commands (along with functions necessary for them to run):
```

Comment:
The module defines the following commands (along with functions necessary for them to run):